doc/chassis/system-ports.md

```diff
@@ -0,0 +1,262 @@
+
+The only restriction on selecting the ifname is that the key "SYSTEM_PORT|asic_name|ifname" must uniquely identify a port on the system.
+
+### 2.6.3 Inband Ports
+
+Inband ports are required to provide control plane connectivity between  forwarding engines. They are connected to the forwarding device local CPU on one side and the internal fabric on the other. 
+
+Every inband port is assigned a System Port ID just like front panel ports which is known to all the forwarding devices. Thus, every inband port is reachable from every forwarding engine.
+
+### 2.6.4 Fabric Ports
+
+The provisioning and management of Fabric ports is outside the scope of this document and will be documented as a separate proposal.
+
+## 2.7 Orchestration Agent
+
+### 2.7.1 System Port Handling
+
+System port configuration is expected to completely static and known at the start of the system.
+
+Based on the state of ‘“connect_chassis_db” in the device metadata in ConfigDB, Orchagent connects to Chassis DB and subscribes to the SYSTEM_PORT table in CHASSIS_DB. It uses this list of system ports from the SYSTEM_PORT table to construct the switch attributes needed by the create_switch SAI API. In a later phase, the system ports could be directly created by making sairedis calls from orchagent. 
+        
+A system port can be used in lieu of a physical port in several SAI API calls as relevant. For example, a system port can be added as a vlan member or be a lag member. To account for these, portsorch will be updated to support sysports. 
+
+
+Portsyncd does not have to support sysports because sysports do not have any associated kernel devices. 
+
+# 3 Testing
+
+Test coverage for the distributed VoQ architecture is achieved by extending the existing virtual switch based SWSS pytest infrastructure.
+
+The distributed switching architecture is represented as multiple VS instances connected with each other and called as Virtual Chassis, where one of the instance plays the role of the SSI and the remaining instances as FSIs. 
+
+## 3.1 Regression Testing
+
+Existing SWSS pytests can be executed against any of the instances in the virtual chassis to ensure that existing SONiC functionality is not affected while operating in a distributed environment.
+
+## 3.2 Distributed VoQ functionality
+
+Additional tests that specifically validate distributed VoQ forwarding functionality run only in the virtual chassis environment. 
+
+### 3.2.3 test_virtual_chassis.py
+
+Is the top level test driver that executes testcases against the virtual chassis. 
+
+Sysport handling is tested by test_chassis_sysport which validates that
+* System ports can be populated in CHASSIS_DB
+* All FSIs can connect to CHASSIS_DB and access sysport state
+* Orchagent programs the correct SAI Redis ASIC_DB state to represent the configured sysport.
+
+# 4 Future Work
```

Comment:
Is multiple SSIs a scenario to consider for the future?

Reply:
Yes - this is an important aspect of future support. We will add a section that captures a short summary.

Reply:
Fixed in latest revision